modules/apps/29-fee/keeper/escrow.go

```diff
@@ -0,0 +1,118 @@
+package keeper
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+
+	"github.com/cosmos/ibc-go/modules/apps/29-fee/types"
+	channeltypes "github.com/cosmos/ibc-go/modules/core/04-channel/types"
+)
+
+// EscrowPacketFee sends the packet fee to the 29-fee module account to hold in escrow
+func (k Keeper) EscrowPacketFee(ctx sdk.Context, refundAcc sdk.AccAddress, identifiedFee types.IdentifiedPacketFee) error {
+	// check if the refund account exists
+	hasRefundAcc := k.authKeeper.GetAccount(ctx, refundAcc)
+	if hasRefundAcc == nil {
+		return sdkerrors.Wrap(types.ErrRefundAccNotFound, fmt.Sprintf("Account with address: %s not found", refundAcc.String()))
+	}
+
+	fees := sdk.Coins{
+		*identifiedFee.Fee.AckFee, *identifiedFee.Fee.ReceiveFee, *identifiedFee.Fee.TimeoutFee,
```

Comment:
THis is wrong anyway. Since i think it will fail if the fees share a denom.

There's a helper function in `sdk/types` to add Coins together correctly.

Check out this file to help you: https://github.com/cosmos/cosmos-sdk/blob/master/types/coin.go

Reply:
> Since i think it will fail if the fees share a denom.

Tests seem to pass with this, interestingly. I was actually just adding the individual fee's to this Coins array to better loop over the fee's and thus reduce code. 

> Check out this file to help you: https://github.com/cosmos/cosmos-sdk/blob/master/types/coin.go

thanks :pray: